Source/WebKit/NetworkProcess/NetworkProcess.messages.in

```diff
@@ -27,6 +27,9 @@ messages -> NetworkProcess LegacyReceiver {
     # Creates a connection for communication with a WebProcess
     CreateNetworkConnectionToWebProcess(bool isServiceWorkerProcess, WebCore::RegistrableDomain registrableDomain)
 
+#if PLATFORM(HAIKU)
+	CreateNetworkConnectionToWebProcessHaiku(bool isServiceWorkerProcess, WebCore::RegistrableDomain registrableDomain,int64_t webPID)
```

Comment:
I felt this was necessary because the usual way other platform does it by creating a socket pair and taking server onto networkprocess and send the client as an attachment to webprocess. So we had to exchange our pid's.